release-please.config.json

```diff
@@ -1,6 +1,4 @@
 {
-  "bootstrap-sha": "79825da91dd20c3be2f47912285e2c977848af32",
```

Comment:
Curious on why we are removing this?

Reply:
It's only used the first time release-please runs. So it is now a noop. [Docs](https://github.com/googleapis/release-please/blob/ace2bd5dc778f83c33ad5dee6807db5d0afdba36/docs/manifest-releaser.md?plain=1#L132)